ui/client/components/graph/NodeUtils.ts

```diff
@@ -200,13 +200,15 @@ class NodeUtils {
 
```

Comment:
@mproch Unfortunately I'am not able to add second edge to any node - first works at glance but second one not

Reply:
Fixed